src/java.desktop/share/classes/javax/swing/text/html/parser/DocumentParser.java

```diff
@@ -25,14 +25,13 @@
 
 package javax.swing.text.html.parser;
 
+import java.io.IOException;
+import java.io.Reader;
+
+import javax.swing.text.ChangedCharSetException;
 import javax.swing.text.SimpleAttributeSet;
-import javax.swing.text.html.HTMLEditorKit;
 import javax.swing.text.html.HTML;
-import javax.swing.text.ChangedCharSetException;
-
-import java.util.*;
-import java.io.*;
-import java.net.*;
+import javax.swing.text.html.HTMLEditorKit;
```

Comment:
Then there's these changes. OK .. too much .. let's just revert all the files that are import only changes.

Reply:
Reverted all import changes.